### PR TITLE
feat(scripts): Improved run-proxies

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
New `--port` option, and better handling of args to allow equality-assignment (e.g. `-p=4321`).
```
Usage: ./scripts/run-proxies.sh [options] [proxy...]
Options:
  -h, --help            Show this help message
  -f, --force           Remove any existing containers every time
  -s, --remove-containers-on-start
                        Remove containers on start
  -x, --remove-containers-on-fail
                        Remove containers on fail
  -i, --interval        Restart interval (default: 1)
  -r, --restart-max-retries
                        Max number of restart retries (default: 3)
  -n, --dry             Dry run
```

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: scripts/run-proxies.sh [hunk 1]
```diff
@@ -91,6 +96,15 @@ parse_cli() {
...
     local value=true
+    # echo "DEBUG: Raw args: '$*'"
+    # Check for equal sign
+    if echo "$arg" | grep -q '='; then
+      # echo "DEBUG: Setting value by equal sign"
+      opt="${arg##*=}"
+      arg="${arg%%=*}"
+      set -- "$arg" "$opt" "${@:2}"
+    fi
+    # echo "DEBUG: arg='$arg' opt='$opt' \$@='$*'"
```
Comment (lodmfjord) [inline scripts/run-proxies.sh:107]: Remove commented out code?

File: scripts/run-proxies.sh [hunk 2]
```diff
@@ -91,6 +96,15 @@ parse_cli() {
     local arg="$1"
     local opt="${2:-}"
     local value=true
+    # echo "DEBUG: Raw args: '$*'"
+    # Check for equal sign
+    if echo "$arg" | grep -q '='; then
+      # echo "DEBUG: Setting value by equal sign"
```
Comment (lodmfjord) [inline scripts/run-proxies.sh:102]: remove commented out code?

File: scripts/run-proxies.sh [hunk 3]
```diff
@@ -91,6 +96,15 @@ parse_cli() {
     local arg="$1"
     local opt="${2:-}"
     local value=true
+    # echo "DEBUG: Raw args: '$*'"
```
Comment (lodmfjord) [inline scripts/run-proxies.sh:99]: remove commented out code?

File: scripts/run-proxies.sh (hunk 1 above)
Comment (lodmfjord) [inline scripts/run-proxies.sh:107]: remove commented out code?

File: scripts/run-proxies.sh [hunk 4]
```diff
@@ -139,24 +155,39 @@ parse_cli() {
...
     shift
   done
+
+  # echo "DEBUG: PROXIES=${PROXIES[*]}"
   # echo "DEBUG: PROXIES=${PROXIES[*]}"
   # echo "DEBUG: ARGS=${ARGS[*]}"
   # echo "DEBUG: REMOVE_CONTAINERS_ON_START=${REMOVE_CONTAINERS_ON_START}"
   # echo "DEBUG: REMOVE_CONTAINERS_ON_FAIL=${REMOVE_CONTAINERS_ON_FAIL}"
   # echo "DEBUG: REMOVE_CONTAINERS_FORCE=${REMOVE_CONTAINERS_FORCE}"
+  # echo "DEBUG: debug exiting" && exit 1
```
Comment (lodmfjord) [inline scripts/run-proxies.sh:176]: remove commented out code? or is this something wew want to keep?

**Review by lodmfjord — APPROVED** [review]
I love what you have done to the place

**Comment by AndesKrrrrrrrrrrr** [comment]
I want thhe commented debug code, for easier debugging later (like I did this time :lady_beetle: )

**Review by snaerseljan — APPROVED** [review]
Nice thank you

**Pull request merged**